src/components/SearchBar.js

```diff
@@ -11,15 +11,22 @@ const SearchBar = ({ value, setValue }) => {
   };
 
   return (
-    <div>
+    <div className="flex mx-2">
       <input
+        className="py-3 border rounded-lg py-2 text-gray-900
+      focus:shadow-outline focus:bg-blue-100 placeholder-gray-400 w-full"
         type="text"
         placeholder="Search list.."
         value={value}
         onChange={handleChange}
       />
       {value && (
```

Comment:
by painting a btn based on value the input element jumps as it centers the element and the center is different if there is only the input element or input element and the button. 
how about doing something like 
`<button
  className={`bg-gray-800 py-2 px-4 rounded-lg text-gray-100 ml-1 shadow-md ${
    !value && 'invisible'
  }`}
`

Reply:
Thanks for the suggestion. But I am not sure if I can follow. I tried your code but nothing seems to change 😕